src/pykeen/evaluation/restricted_evaluator.py

```diff
@@ -0,0 +1,157 @@
+import torch
```

Comment:
Maybe we can bring this into `master` in an earlier PR?

Reply:
Agreed.